cfy_manager/components/composer/composer.py

```diff
@@ -253,8 +240,8 @@ def remove(self):
 
     def start(self):
         logger.notice('Starting Cloudify Composer...')
-        systemd.start(COMPOSER)
-        self._verify_composer_alive()
+        self._run_db_migrate()
```

Comment:
Is this safe to do on a cfy_manager restart? I'm assuming the migrate will do nothing if it's already up to date?

Reply:
turns out it is. Restart was super slow because it was restarting everything 3 times, but #778 fixes that